recipe/meta.yaml

```diff
@@ -14,12 +14,13 @@ package:
 
 source:
   url: https://github.com/Unidata/netcdf-c/archive/v{{ version }}.tar.gz
```

Comment:
If there are formal release tarballs then you might change this URL to reflect that, then the bot most likely wouldn't find the tag without a release.

Reply:
It is important to note that, for general purposes, a tag **is** a release. The GitHub release is just HTML sugar around it. An rc/dev/alpha/beta tag would be an "incomplete" release.

Reply:
I strongly second what @ocefpaf said.

Reply:
For our part, once a release is 'complete', be it pre-release, RC or full-release, it becomes a fixed release and cannot be modified/changed without playing merry havoc with packaging systems and checksums.  The idea of having the tag, without the formal release 'sugar' surrounding it, is that it may still be subject to change.  If this proves infeasible due to automated scraping/processing of *any* tag, then we'll adopt a different approach.

Reply:
One can also change the GitHub release BTW. Your workflow is fine for you, just note standard OSS and we will have to work around it.

Reply:
@ocefpaf we *can*, but the last time we did that (4, 5 years ago?) we heard about it from the Redhat maintainers who suddenly had to figure out why checksums weren't matching up any longer.  It was a big, messy knot that our approach is meant to avoid.

Reply:
I think the point here is that *all github releases and tags* are malleable in principle, not that one should be changing one over the other.

Reply:
Yep. There is virtually no difference in that regard.

Reply:
@xylar Do we need to point this url to the github release per the convo above?

Reply:
even if this is the right tag, it might be good to point it there anyways.

Reply:
No, a tag and a release have the exact same URL as I understand it.  There's no difference.  The URL for the tag created 4 days ago is the one for the release.

Reply:
If you go to: https://github.com/Unidata/netcdf-c/releases
You will get this link from the 4.7.3 release
https://github.com/Unidata/netcdf-c/archive/v4.7.3.tar.gz
This is exactly what we're using above.

Reply:
I also did a local test build to verify that the sha256 hasn't changed and it hadn't.